client/src/i18n/en/report.json

```diff
@@ -158,8 +158,8 @@
     },
 
     "STOCK_MOVEMENT_REPORT" : {
-      "TITLE" : "Stock Movement Graphical report",
-      "DESCRIPTION" : ""
+      "TITLE" : "Stock Movement Graph",
+      "DESCRIPTION" : "This report displays in graphic form the various stock movements made during a period"
```

Comment:
This would be better written as "This report displays a graph of stock movements made during a period".

"in graphic form" in English typically means something is very detailed or very vulgar. 😅